src/components/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -0,0 +1,300 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import EmailTemplate from './EmailTemplate';
+import EmailHistory from './EmailHistory';
+import axios from 'axios';
+import CheckButton from './CheckButton';
+import { EMAILAPI_URL } from '../../../../constants';
+
+const Emails: React.FC = () => {
+  const [mentees, setMentees] = useState([]);
+  const [selectedMentees, setSelectedMentees] = useState<string[]>([]);
+  const [subject, setSubject] = useState('');
+  const [body, setBody] = useState('');
+  const [view, setView] = useState('sent');
+  const [showRefresh, setShowRefresh] = useState(false);
+  const [apiStatus, setApiStatus] = useState('Checking...');
+  const [emails, setEmails] = useState([]);
+  const [refreshCount, setRefreshCount] = useState(0);
+  const [isRefreshing, setIsRefreshing] = useState(true);
+  const [select, setSelect] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [message, setMessage] = useState('');
+
+  const fetchEmails = useCallback(() => {
+    axios
+      .get(`${EMAILAPI_URL}/api/v1/sent`)
+      .then((response) => {
+        setEmails(response.data.recipient);
+      })
+      .catch((error) => {
+        console.error('Error:', error);
+      });
+  }, []);
+
+  console.log(EMAILAPI_URL);
+  useEffect(fetchEmails, []);
+
+  const checkApiStatus = () => {
+    setIsLoading(true);
+    axios
+      .get(`${EMAILAPI_URL}/api/v1/healthcheck`)
+      .then((response) => {
+        setTimeout(() => {
+          if (response.data.status === 'available') {
+            setApiStatus('Connected');
+          } else {
+            setApiStatus('Disconnected');
+          }
+        }, 1000);
+      })
+      .catch((error) => {
+        setTimeout(() => {
+          setApiStatus('Error');
+          setMessage(`Error : ${(error as Error).message}`);
+        }, 1000);
+      })
+      .finally(() => {
+        setTimeout(() => {
+          setIsLoading(false);
+        }, 1000);
+      });
+  };
+  useEffect(() => {
+    checkApiStatus();
+  }, []);
+
+  const handleFormSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const emailData = {
+      sender: 'mayuraalahakoon@gmail.com',
```

Comment:
```suggestion
      sender: 'mayuraalahakoon@gmail.com',
```
Move this to the constants.

Reply:
i'll update this